client/src/main/java/no/nordicsemi/android/kotlin/ble/client/main/service/ClientBleGattCharacteristic.kt

```diff
@@ -207,16 +233,27 @@ class ClientBleGattCharacteristic internal constructor(
         mutex.lock()
         val stacktrace = Exception() //Helper exception to display valid stacktrace.
         return suspendCoroutine { continuation ->
-            logger.log(Log.DEBUG, "Write to characteristic - start, uuid: $uuid, value: $value, type: $writeType")
+            logger.log(
+                Log.DEBUG,
+                "Write to characteristic - start, uuid: $uuid, value: $value, type: $writeType"
```

Comment:
Is the value here printed as hex or an if of a reference?

Reply:
value is of type DataByteArray which overrides toString() and uses hex as an output